tests/forte/data/audio_annotation_test.py

```diff
@@ -192,19 +228,24 @@ def test_audio_annotation(self):
 
                 for audio_utter in audio_utters:
                     configs: Dict = self._test_configs[audio_utter.speaker]
-                    self.assertTrue(array_equal(
-                        audio_utter.audio,
-                        pack.audio[configs["begin"]:configs["end"]]
-                    ))
+                    self.assertTrue(
+                        array_equal(
+                            audio_utter.audio,
+                            pack.get_payload_data_at(Modality.Audio, 0)[
```

Comment:
`pack.audio` is enough. In PR to #841, you can try to add unit tests which involve multiple audio payloads.

Reply:
`pack.audio` is removed in this PR.